charts/calico/templates/calico-node-rbac.yaml

```diff
@@ -242,3 +284,15 @@ subjects:
   name: calico-node
   namespace: kube-system
 {{- end }}
+kind: ClusterRoleBinding
```

Comment:
I think this can be a `RoleBinding` since it's only for a single serviceaccount in one namespace.

Reply:
In your other PR, this is a RoleBinding: https://github.com/tigera/operator/pull/2393/files#diff-d4d31d15e69aa9a0bad7776e758dfbd50b8e0e1a969fd16719f56c470cac4e7bR316